### ISSUE TITLE
Minimise occurrences of the log: `[WARN] client: error fetching host disk usage stats`

### ISSUE THREAD

**Issue by hmalphettes** [hashicorp/nomad#2700]
### Nomad version
`Nomad v0.5.6`

### Operating system and Environment details
`CentOS Linux release 7.1`

### Issue
Every few seconds, nomad logs
```
[WARN] client: error fetching host disk usage stats for /media/123: permission denied
```
Here is the PR where the log was introduced: https://github.com/hashicorp/nomad/pull/2090

### Reproduction steps
Run nomad as a non privileged user.
Have a mounted partition where the user running nomad does not have read access.
Let me know if you need more specific steps: I did not setup that server.

Would you consider issuing the warning just once?
Should we in fact ignore a `permission denied` error all together: after all it makes sense that a non-privileged user does not monitor a partition it has no access to?

I can attempt to contribute a PR if you want.
Let me know which way to go in that case.

Running nomad as a non-privileged user paired with the raw-exec driver is a less exciting setup nowadays. But it is a requirement for a vendor in a financial institution where security compliance relies entirely on running as a non-privileged user.

Many thanks for your attention!

**Comment by dadgar** [comment]
Hey @hmalphettes 

I would love a PR! It is probably worth logging the error only once on failure. We follow the pattern in the fingerprinters: https://github.com/hashicorp/nomad/blob/master/client/fingerprint/consul.go#L24

You can see it only logs failures on state changes!

**Comment by hmalphettes** [comment]
@dadgar thanks for the pointer. I'll find some time to do it then. If someone beats me to it, all the better.
The rest of the stack is working so well that this remains a detail... and we are busy until we go in production.

**Comment by tgross** [comment]
Doing some issue cleanup. Looks like this was fixed in 7a84cbe02a32037a72a3b855236c4012f9a8e78f which shipped in 0.6.1
